saidump/saidump.cpp

```diff
@@ -399,7 +418,159 @@ void dumpGraph(const TableDump& td)
     std::cout << "}" << std::endl;
 }
 
-int main(int argc, char ** argv)
+/*
+ preprocess the input json file to make sure it's a valid json file.
+*/
+int pre_process_file(string file_name)
```

Comment:
rename preProcessFile, and make function static

Reply:
Fix it.